File: src/scanner/code/__fixtures__/python/safe.py
"""Test fixture: Python code with quantum-safe cryptography."""
from cryptography.hazmat.primitives import hashes
import hashlib

# SHA-256 (SAFE)
sha256_hash = hashlib.sha256(b"data")

# SHA-384 (SAFE)
sha384_hash = hashlib.sha384(b"data")

# SHA-512 (SAFE)
sha512_hash = hashlib.sha512(b"data")

# SHA-3 (SAFE)
sha3_hash = hashlib.sha3_256(b"data")

# HMAC-SHA256 (SAFE - symmetric)
import hmac
mac = hmac.new(b"secret", b"data", hashlib.sha256)

File: src/scanner/code/__fixtures__/javascript/safe.js
/**
 * Test fixture: JavaScript code with quantum-safe cryptography.
 */
const crypto = require('crypto');
const jwt = require('jsonwebtoken');

// AES-256 (SAFE)
const cipher = crypto.createCipheriv('aes-256-gcm', key32, iv);

// SHA-256 (SAFE)
const hash = crypto.createHash('sha256');

// SHA-384 (SAFE)
const hash384 = crypto.createHash('sha384');

// HMAC-SHA256 (SAFE)
const hmac = crypto.createHmac('sha256', secret);

// JWT with HMAC (SAFE - symmetric)
const token = jwt.sign({ sub: '1234' }, 'shared-secret', { algorithm: 'HS256' });
